projects/packages/search/src/class-template-tags.php

```diff
@@ -1,33 +1,34 @@
 <?php
 /**
- * Jetpack Search: Jetpack_Search_Template_Tags class
+ * Template tags class used primarily for rendering widget-related HTML.
  *
- * @package    Jetpack
- * @subpackage Jetpack Search
- * @since      5.8.0
+ * @package    automattic/jetpack-search
  */
 
-use Automattic\Jetpack\Search\Helper;
+namespace Automattic\Jetpack\Search;
+
+use Automattic\Jetpack\Search\Classic_Search as Jetpack_Search;
```

Comment:
I couldn't  find `Classic_Search` anywhere, which is in the big migration branch.

Reply:
Ah, good catch. This change wasn't intended for this PR.